scripts/install/jellyfin.sh

```diff
@@ -108,219 +76,56 @@ CONFIG
   <BaseUrl />
-  <UICulture>en-US</UICulture>
-  <SaveMetadataHidden>false</SaveMetadataHidden>
-  <ContentTypes />
-  <RemoteClientBitrateLimit>0</RemoteClientBitrateLimit>
-  <EnableFolderView>false</EnableFolderView>
-  <EnableGroupingIntoCollections>false</EnableGroupingIntoCollections>
-  <DisplaySpecialsWithinSeasons>true</DisplaySpecialsWithinSeasons>
-  <LocalNetworkSubnets />
   <LocalNetworkAddresses>
     <string>0.0.0.0</string>
   </LocalNetworkAddresses>
-  <CodecsUsed />
-  <IgnoreVirtualInterfaces>false</IgnoreVirtualInterfaces>
-  <EnableExternalContentInSuggestions>true</EnableExternalContentInSuggestions>
-  <RequireHttps>false</RequireHttps>
-  <IsBehindProxy>false</IsBehindProxy>
-  <EnableNewOmdbSupport>false</EnableNewOmdbSupport>
-  <RemoteIPFilter />
-  <IsRemoteIPFilterBlacklist>false</IsRemoteIPFilterBlacklist>
-  <ImageExtractionTimeoutMs>0</ImageExtractionTimeoutMs>
-  <PathSubstitutions />
-  <EnableSimpleArtistDetection>true</EnableSimpleArtistDetection>
-  <UninstalledPlugins />
 </ServerConfiguration>
 CONFIG
 #
-# Create the service file that will start and stop jellyfin.
-cat > "/etc/systemd/system/jellyfin.service" <<-SERVICE
-[Unit]
-Description=Jellyfin
-After=network.target
-
-[Service]
-User=${username}
-Group=${username}
-UMask=002
-
-Type=simple
-WorkingDirectory=$install_dir
-ExecStart=$install_dir/jellyfin -d /home/${username}/.config/Jellyfin
-TimeoutStopSec=20
-KillMode=process
-Restart=always
-RestartSec=2
-
-[Install]
-WantedBy=multi-user.target
-SERVICE
-#
 # Configure the nginx proxypass using positional parameters.
 if [[ -f /install/.nginx.lock ]]; then
     bash "/usr/local/bin/swizzin/nginx/jellyfin.sh" "${app_port_http}" "${app_port_https}"
     systemctl reload nginx
 fi
 #
 # Set the correct and required permissions of any directories we created or modified.
-chown "${username}.${username}" -R "$install_dir"
-chown "${username}.${username}" -R "$install_ffmpeg"
 chown "${username}.${username}" -R "/home/${username}/.config"
 chown "${username}.${username}" -R "/home/${username}/.ssl"
+chmod -R g+rw "/home/${username}/.config/Jellyfin"
+chmod -R g+r "/home/${username}/.ssl"
+#
+# COnfigure the jellyfin service.
+#
+systemctl stop "jellyfin.service" >> /dev/null 2>&1
+#
+cat > "/etc/systemd/system/jellyfin.service.d/jellyfin.service.conf" <<-CONFIG
+# Jellyfin systemd configuration options
+
+# Use this file to override the user or environment file location.
+
+[Service]
+#User = jellyfin
+EnvironmentFile = /etc/default/jellyfin-swizzin
+CONFIG
+#
+cp -r /etc/default/jellyfin /etc/default/jellyfin-swizzin
+sed -r "s|JELLYFIN_CONFIG_DIR=(.*)|JELLYFIN_CONFIG_DIR=\"/home/${username}/.config/Jellyfin/config\"|g" -i /etc/default/jellyfin-swizzin
+#
+usermod -a -G ${username} jellyfin
 #
-# Enable and start the jellyfin service.
 systemctl daemon-reload
-systemctl enable --now "jellyfin.service" >> /dev/null 2>&1
+systemctl start "jellyfin.service" >> /dev/null 2>&1
```

Comment:
Should this not be `enable --now` in case this was a new install?

Reply:
I am not installing or starting the service. apt is. It's already enabled.

Reply:
oooh gotcha.

Reply:
/resolved